### ISSUE TITLE
[OH FUCK] Truth Dust is not removed from UI when fully consumed

### ISSUE THREAD

**Issue by DeckardCainRPC** [ChalkyBrush/roshpit-bug-tracker#734]
STEPS TO REPRODUCE:
1. get Truth Dust
2. use it all up
3. its icon is now permanently resident in your inventory, obscuring any items that you might place in those slots later. this persists for the entire game session.

**Comment by redledcat** [comment]
Correct, but you missed important fact - only happens after Redfall run, or other multiple runs in single session.

**Comment by DeckardCainRPC** [comment]
really? i was not aware of that

**Comment by redledcat** [comment]
![20180913202255_1](https://user-images.githubusercontent.com/32621322/45504881-c18b2f80-b793-11e8-817d-51a52a0d51f1.jpg)
![20180913202541_1](https://user-images.githubusercontent.com/32621322/45504884-c354f300-b793-11e8-9de3-b94a82babccd.jpg)
here, dust in game straight after redfall, and after crash

**Comment by redledcat** [comment]
```
"OnSpellStart"
		{
			"SpendCharge"
			{}
			"RunScript"
			{
				"ScriptFile"	"worlds/arena/pit_of_trials/pit_of_trials_ai"
				"Function"		"use_truth_dust"
			}
		}
```
maybe SpendCharge should be after script?
